app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :users
+  belongs_to :posts
+
+  def update_likes_counter
+    post.increment!(:likesCounter)
+  end
+end
```

Comment:
-  Just like you did in the Post model, please use the `after_save` method to call this function, this would ensure that the likesCounter is only updated when the comment is saved in the database 😄